src/Raven.Server/Smuggler/Migration/MigrationConfiguration.cs

```diff
@@ -1,8 +1,22 @@
 using System.Collections.Generic;
 using Raven.Client.Documents.Smuggler;
+using Sparrow.Json;
 
 namespace Raven.Server.Smuggler.Migration
 {
+    public class MigrationConfiguration
+    {
+        public string DatabaseTypeName { get; set; }
```

Comment:
enum?

Reply:
Still waiting

Reply:
this is "mongodb" or "cosmosdb"

Reply:
But can be more in the future. Enum would be better?